fuzz/Cargo.toml

```diff
@@ -8,7 +8,7 @@ edition = "2021"
 cargo-fuzz = true
 
 [dependencies]
-libfuzzer-sys = { git = "https://github.com/AFLplusplus/LibAFL.git", tag = "0.13.0", package = "libafl_libfuzzer" }
+libfuzzer-sys = { git = "https://github.com/AFLplusplus/LibAFL.git", tag = "0.14.1", package = "libafl_libfuzzer" }
```

Comment:
Actually, we should be able to use https://crates.io/crates/libafllibafl_libfuzzer?